runtime/vm/threadhelp.cpp

```diff
@@ -124,6 +125,15 @@ monitorWaitImpl(J9VMThread *vmThread, j9object_t object, I_64 millis, I_32 nanos
 	} else {
 		rc = omrthread_monitor_wait_timed(monitor, millis, nanos);
 	}
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
instead we can just throw an exception in the wait INL/fastJNI implementations so we never end up here

Reply:
This is not required after merging of #14583.